opm/input/eclipse/Schedule/Well/Well.hpp

```diff
@@ -168,8 +168,15 @@ class Well {
         static WellInjectionProperties serializationTestObject();
 
         void handleWELTARG(WELTARGCMode cmode, const UDAValue& new_arg, double SIFactorP);
-        void handleWCONINJE(const DeckRecord& record, bool availableForGroupControl, const std::string& well_name);
-        void handleWCONINJH(const DeckRecord& record, const bool is_producer, const std::string& well_name, const KeywordLocation& loc);
+        void handleWCONINJE(const DeckRecord& record,
+                            const double bhp_def,
+                            bool availableForGroupControl,
+                            const std::string& well_name);
+        void handleWCONINJH(const DeckRecord& record,
+                            const double bhp_def,
+                            const bool is_producer,
+                            const std::string& well_name,
+                            const KeywordLocation& loc);
```

Comment:
Maybe you could add a comment explaining the expected unit conventions for `bhp_def` here&ndash;e.g., input units or SI units as appropriate?

Reply:
Yeah, it's certainly a bit meh.

The difference is due to the fact that limits are stored as raw numbers in SI, while targets are stored as UDAs with pressure dimension. It's actually consistent though in that we always pass SI units around and eat a few extra conversions back from SI to pressure measure where we update UDAs. Maybe I did not end up with the most readable approach, I can try to reiterate.

Reply:
hopefully it's better now.